core/src/test/java/org/testcontainers/utility/DockerImageNameTest.java

```diff
@@ -0,0 +1,18 @@
+package org.testcontainers.utility;
+
+import org.junit.Test;
+
+public class DockerImageNameTest {
+
+    @Test
+    public void validNames() {
+        DockerImageName.validate("myname:latest"); // no repo
+        DockerImageName.validate("repo/my-name:1.0"); // no repo
+        DockerImageName.validate("repo.foo.com:1234/my-name:1.0"); // no repo
+    }
```

Comment:
Cool, thanks for the confirmation. @asafm please could you add this test if you have a chance? I suspect the current implementation will be fine, so it's just to make sure this is tested.